engine/table/src/main/java/io/deephaven/engine/table/impl/NaturalJoinHelper.java

```diff
@@ -65,8 +65,9 @@ private static Table naturalJoinInternal(QueryTable leftTable, QueryTable rightT
 
                 final QueryTable result = makeResult(leftTable, rightTable, columnsToAdd, rowRedirection, true);
 
-                leftTable.listenForUpdates(new LeftTickingListener(bucketingContext.listenerDescription, columnsToMatch,
-                        columnsToAdd, leftTable, result, rowRedirection, jsm, bucketingContext.leftSources));
+                leftTable
```

Comment:
The formatting changes from spotless in this file kind of stink. It would be nice if we beautified things a little.

Reply:
I made this and one similarly-ugly line nicer (w/ linebreaks between arguments)